v3/handwired/dactyl_maximus/dactyl_maximus.json

```diff
@@ -0,0 +1,121 @@
+{
+  "name": "Dactyl Maximus",
+  "vendorId": "0x444D",
+  "productId": "0x3536",
```

Comment:
![image](https://github.com/the-via/keyboards/assets/17491233/02bc9fd2-0af4-4758-9fc7-59608dab1ac1)

github actions is flagging this as a duplicate VID/PID combo.

Reply:
Created [corrective PR](https://github.com/qmk/qmk_firmware/pull/21666) in qmk/qmk_firmware for `dactyl_maximus` VID and PID